src/Controls/samples/Controls.Sample/Pages/MainPage.cs

```diff
@@ -83,10 +83,10 @@ void SetupMauiLayout()
 
 			verticalStack.Add(horizontalStack);
 
-			verticalStack.Add(new CheckBox());
-			verticalStack.Add(new CheckBox { BackgroundColor = Color.LightPink });
-			verticalStack.Add(new CheckBox { IsChecked = true, Color = Color.Aquamarine });
-
+			verticalStack.Add(new Microsoft.Maui.Controls.CheckBox());
```

Comment:
I think this is because there is the native version. Might have to use a different name on that one.

Reply:
Fixed.